### PR TITLE
add archive message + fix deadname <3

### PR DESCRIPTION
**Pull request by itsmeimtom**

### CODE REVIEW

**Comment by itsmeimtom** [comment]
failed? ok maybe that script does need to be elsewhere...

**Comment by lewisakura** [comment]
> failed? ok maybe that script does need to be elsewhere...

put the alert in an `onMount` in the module script

**Comment by itsmeimtom** [comment]
> put the alert in an `onMount` in the module script

lewis this may as well have been written in hieroglyphics 😭

one of you two will have to make it properly (and feel free to change the wording and etc)

**Comment by Gxorge** [comment]
ill do it.

File: src/routes/index.svelte [hunk 1]
```diff
@@ -97,6 +97,15 @@
 </header>
 
 <article>
+
+	<figure class="message">
+		<p>
+			<!-- sorry lewis this is your problem -->
```
Comment (itsmeimtom) [inline src/routes/index.svelte:103]: What's this comment about?

**Review by lewisakura — APPROVED** [review]

**Review by itsmeimtom — COMMENTED** [review]
Remove the comment that no longer makes sense and it looks good (sadly I cannot approve my own PR)

File: src/routes/index.svelte (hunk 1 above)
Comment (Gxorge) [reply src/routes/index.svelte:103]: i've got no idea i just copy and pased from before

**Comment by Gxorge** [comment]
oh well its done now lol

**Comment by itsmeimtom** [comment]
wow lewis can't believe you would merge before waiting for me

**Pull request merged**